.github/workflows/README.md

```diff
@@ -0,0 +1,282 @@
+# Workflows
+
+## `add-to-project.yaml`
+
+<p>
+This workflow adds pull requests and issues to a specific GitHub project board when they are opened.
+</p>
+
+- ## Trigger
+The workflow is triggered by the following events:
+
+> - A pull request being opened.
+
+> - An issue being opened.
+
+- ## Jobs
+The workflow has a single job named `add-to-project`. The following step is executed in this job:
+> - The `add-to-project` job uses the `actions/add-to-project@v0.4.0` action to add pull requests and issues to a specific project board. The `with` parameters are `project-url` and `github-token`, which specify the URL of the project board and the GitHub token used to authenticate the action.
+
+For more details, please refer to the [add-to-project.yaml](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/.github/workflows/add-to-project.yaml) file in the repository.
+
+---
+## `benchmark.yaml`
+
+<p>
+This workflow runs the MiGraphX performance benchmarks and generates comparison reports by comparing the results with the reference data.
+</p>
+
+- ## Trigger
+> The workflow is triggered manually through the "Run workflow" button in the Actions tab of the repository and it will run reusable workflow [benchmarks.yml (archived)](https://github.com/ROCmSoftwarePlatform/actions/blob/main/.github/workflows/benchmarks.yml)
+
+- ## Input Parameters
+The workflow uses the following input parameters:
+
+> - `rocm_version`: the version of ROCm to use for running the benchmarks.
+
+> - `script_repo`: the repository containing the benchmark scripts.
+
+> - `result_path`: the path where the benchmark results will be stored.
+
+> - `result_repo`: the repository where the benchmark results will be pushed for comparison.
+
+For more details, please refer to the [benchmark.yaml](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/.github/workflows/benchmark.yaml) file in the repository.
+
+---
+
+## `ci.yaml`
+
+<p>
+Overall, this workflow automates the process of building and testing the AMDMIGraphX project across multiple platforms and versions.
+</p>
+
+- ## Trigger
+The workflow is triggered by the following events:
+
+> - A pull request being opened, synchronized or closed.
+
+> - On push to the `develop`, `master`, and `release/**` branches.
+
+- ## Jobs
+The following jobs are executed in the workflow:
+> - `cancel`: This job is responsible for canceling any previous runs of the workflow that may still be running. It runs on an `ubuntu-latest` runner and uses the `styfle/cancel-workflow-action` action to cancel any previous runs of the workflow.
+
+> - `tidy`: It runs on an `ubuntu-20.04` runner and runs `clang-tidy` for the codebase in a Docker container with the MIGraphX build environment.
+>The following steps are executed in this job:
+>   - `Free space`: step deletes some directories that may take up space on the runner.
```

Comment:
remove these sub bullets.  They could change and make this readme stale.  Same for the other jobs listed below

Reply:
Done.